cpp/daal/src/algorithms/kmeans/oneapi/cl_kernels/kmeans_cl_kernels_distr.cl

```diff
@@ -21,15 +21,15 @@
 //--
```

Comment:
File naming: src/algorithms/**kmeans**/oneapi/**cl_kernels**/**kmeans_cl_kernels**_distr.cl
Why do you repeat yourself?

Reply:
It's original naming. I just follow it.

Reply:
It's minor - but it'd be good if you don't like this name to rename the file.